### PR TITLE
[Fix] Fix symlink failure on Windows

### PR DESCRIPTION
**Pull request by Junjun2016**
Fix symlink failure on Windows.
Reference: https://github.com/open-mmlab/mmdetection/pull/6482.

### CODE REVIEW

**Comment by RockeyCoss** [comment]
Please merge the master branch into your branch, thank you.

File: setup.py [hunk 1]
```diff
@@ -110,9 +111,11 @@ def add_mim_extension():
     if 'develop' in sys.argv:
         # installed by `pip install -e .`
         mode = 'symlink'
```
Comment (xvjiarui) [inline setup.py:113]: Should also handle `develop` mode?

Comment (Junjun2016) [reply setup.py:113]: Yes, resolved.

**Comment by Junjun2016** [comment]
Hi @xvjiarui 
Resolve all comments.

File: setup.py [hunk 2]
```diff
@@ -109,10 +110,16 @@ def add_mim_extension():
...
         # installed by `pip install -e .`
-        mode = 'symlink'
-    elif 'sdist' in sys.argv or 'bdist_wheel' in sys.argv:
+        if platform.system() == 'Windows':
+            # set `copy` mode here since symlink fails on Windows.
+            mode = 'copy'
+        else:
+            mode = 'symlink'
+    elif 'sdist' in sys.argv or 'bdist_wheel' in sys.argv or \
+            platform.system() == 'Windows':
```
Comment (zhouzaida) [inline setup.py:119]: please refer to https://github.com/open-mmlab/mmflow/pull/37

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**